climate-monitoring-core/src/main/java/climatemonitoring/core/View.java

```diff
@@ -1,6 +1,46 @@
+/*
+
+Alessandro della Frattina 753073 VA
+Cristian Capiferri 752918 VA
+Francesco Lops 753175 VA
+Dariia Sniezhko 753057 VA
+
+*/
+
 package climatemonitoring.core;
 
-public class View {
+/**
+ * The View class manages the state of a view and handles rendering operations based on the configuration (headless or GUI)
+ * @author francescolops
+ * @version 1.0-SNAPSHOT
+ */
+public class View{
 
+	/**
+	 * Sets the view state
+	 * @param state
+	 */
+	public void setState(ViewState state){
+		
+		m_State = state;
+	}
 	
+	/**
+	 * Code that runs in headless configuration only
+	 * @param args
```

Comment:
You should put a description if you use the tag `@param`, but if nothing comes to your mind, then remove this from the javadoc